scripts/travis_ubuntu.sh

```diff
@@ -3,10 +3,17 @@
 ## -x be verbose; write what we are doing, as we do it
 set -ex
 
+# for non-LTS && EOL'ed ubuntu, the mirror url must be modified to run `apt update`
+cmd=ls
+if [ $2 = ":eoan" ]; then
+    cmd="sed -i 's/\(security\|archive\).ubuntu/old-releases.ubuntu/g' /etc/apt/sources.list"
+fi 
+
 sudo docker pull "$1$2"                                                                                           \
 &&                                                                                                                \
 sudo docker create --name mobydick "$1$2" /bin/bash -c                                                            \
-"apt-get    update                                                                                             && \
+"${cmd} &&
```

Comment:
@RyanBernX : before merging, can you add a `\` at the end of line ? Not sure how this works without it. Thanks for your help !

Reply:
@fghoussen Added. BTW I'll skip this CI since https://github.com/opencollab/arpack-ng/pull/299/commits/3f66e11946271df3f4420e4a01c6f30e06958f20 has been confirmed to work.